wordpress-mobile/gutenberg-mobile
# [Fix] Columns block renders more than two columns in a row when launching in landscape mode

Fixes #2405 
Gutenberg PR - https://github.com/WordPress/gutenberg/pull/23305

To test:
- Make sure that your editor has more than two columns in a columns block
- Turn the device or emulator in landscape mode
- Start the Gutenberg-Mobile app or reload
- Observe that max two columns are rendered in each row
- Switch to the portrait mode
- Only one column should be rendered in a row

PR submission checklist:
